### DISCUSSION TITLE
multipule source folders?

### DISCUSSION THREAD

**Discussion by Sleuth56** [cupcakearmy/autorestic#111] [Q&A] [answered]
I want to be able to backup multiple folders at once. Is this possible?
Currently I'm using `from:` with multiple folders but that gives a config error. 
Is there some other way to do this?

**Comment by cupcakearmy** [comment, marked as answer]
Currently not possible.
https://github.com/cupcakearmy/autorestic/issues/71
There is an open issue but it'wont happen soon i think
